hack/tools.go

```diff
@@ -37,5 +37,6 @@ import (
 	_ "knative.dev/networking/config"
 
 	_ "github.com/mikefarah/yq/v3"
+	_ "github.com/projectcontour/contour-operator/cmd"
```

Comment:
Do you think we should still be vendoring and building ourselves?

Reply:
If so then the operator should probably use the vendored contour

Reply:
I wondered that we should still be building it or not. I think we should not build operator by ourselvs.
Thank you, updated.